docs/proposals/registry/index-server-REST-API.md

```diff
@@ -0,0 +1,239 @@
+# Index server REST APIs
+This documentation explains how to use the index server REST APIs
+
+## Gets registry index file  
```

Comment:
Can we add a description for each entry to briefly explain what each API is used for?

Reply:
The description is similar to the heading, let me try to add more details to it.